### ISSUE TITLE
Can not see the full debug log on iOS?

### ISSUE THREAD

**Issue by lan-indiez** [appmetrica/appmetrica-unity-plugin#2]
I have implemented your SDK for my iOS app and used the function `ReportEvent` with `jsonValue` to track my events.
However, when I tested our game with Xcode or Console on macOS, I just only saw the `eventName`, and didn't see the `jsonValue` that following

How can I fix this?

**Comment by NesterovichAlexey** [comment]
Hello! Currently, only the `eventName` is logged. In future releases, we will also log `jsonValue`.

**Comment by NesterovichAlexey** [comment]
[Internal ticket](https://nda.ya.ru/t/zlwWgISw75hwEj)

**Comment by lan-indiez** [comment]
> Hello! Currently, only the `eventName` is logged. In future releases, we will also log `jsonValue`.

Hi @NesterovichAlexey thank you for your reply, can you give me the est time for the new release to include the log `jsonValue`?
This log is really important with us to confirm all event is recorded perfectly

**Comment by NesterovichAlexey** [comment]
> Hi @NesterovichAlexey thank you for your reply, can you give me the est time for the new release to include the log jsonValue?

Hi! I can't tell you when this problem will be fixed. I forwarded the problem to the AppMetrica SDK iOS team and it needs to be fixed there. Once they fix it I will update the ios sdk version in the unity plugin. 
You can view events in the AppMetrica web interface https://appmetrica.io/docs/en/data-collection/about-events.

**Comment by lan-indiez** [comment]
> > Hi @NesterovichAlexey thank you for your reply, can you give me the est time for the new release to include the log jsonValue?
> 
> Hi! I can't tell you when this problem will be fixed. I forwarded the problem to the AppMetrica SDK iOS team and it needs to be fixed there. Once they fix it I will update the ios sdk version in the unity plugin. You can view events in the AppMetrica web interface https://appmetrica.io/docs/en/data-collection/about-events.

Hi @NesterovichAlexey do you have any updates about this? I'm really need this for debug our events

**Comment by NesterovichAlexey** [comment]
Hi @lan-indiez !
It's still not fixed. But you can add your log before calling `AppMetrica.ReportEvent`.

**Comment by Sergey70** [comment]
Logging JSON parameters for iOS SDK has been added and will be included in the next release.

**Comment by lan-indiez** [comment]
> This has been fixed in version [6.3.0](https://github.com/appmetrica/appmetrica-unity-plugin/releases/tag/v6.3.0).

Hi @NesterovichAlexey , I'm using your version 6.3.0 and I did not receive any event param log in console. Here is the error I received:

```
    "4K5NP4SUYH.com.indiez.nonogram"
)" UserInfo={numberOfErrorsDeep=0, NSDescription=Client explicitly specifies access group 4K5NP4SUYH.io.appmetrica but is only entitled for (
    "4K5NP4SUYH.com.indiez.nonogram"
)}
Nov  7 12:01:10 Sandbox-03s-iPhone PixelArt(AppMetricaLog)[2821] <Error>: -[AMAKeychain dataForKey:error:]:182 Failed to retrieve data for key AMAKeychainAvailabilityCheckObjectKey, osstatus -34018
Nov  7 12:01:10 Sandbox-03s-iPhone PixelArt(AppMetricaLog)[2821] <Error>: +[AMAErrorUtilities fillError:withError:]:18 Error Domain=kAMAKeychainErrorDomain Code=2 "(null)" UserInfo={kAMAKeychainErrorKeyCode=-34018}
Nov  7 12:01:10 Sandbox-03s-iPhone securityd[105] <Notice>: PixelArt[2821]/1#4 LF=0 add Error Domain=NSOSStatusErrorDomain Code=-34018 "Client explicitly specifies access group 4K5NP4SUYH.io.appmetrica but is only entitled for (
    "4K5NP4SUYH.com.indiez.nonogram"
)" UserInfo={numberOfErrorsDeep=0, NSDescription=Client explicitly specifies access group 4K5NP4SUYH.io.appmetrica but is only entitled for (
    "4K5NP4SUYH.com.indiez.nonogram"
)}
Nov  7 12:01:10 Sandbox-03s-iPhone PixelArt(AppMetricaLog)[2821] <Error>: -[AMAKeychain addData:forKey:error:]:150 Failed to add object for key AMAKeychainAvailabilityCheckObjectKey with osstatus -34018
Nov  7 12:01:10 Sandbox-03s-iPhone PixelArt(AppMetricaLog)[2821] <Error>: -[AMADatabaseIntegrityProcessor checkIntegrityIssuesForDatabase:report:]:50 DB integrity check found issues: (
)
Nov  7 12:01:10 Sandbox-03s-iPhone PixelArt(UnityFramework)[2821] <Notice>: AppMetricaController-> _DebugDeviceID
Nov  7 12:01:10 Sandbox-03s-iPhone PixelArt(AppMetricaLog)[2821] <Error>: -[AMADatabaseIntegrityProcessor checkIntegrityIssuesForDatabase:report:]:50 DB integrity check found issues: (
)
Nov  7 12:01:10 Sandbox-03s-iPhone PixelArt(UnityFramework)[2821] <Notice>: AppMetricaController-> _DebugDeviceID: result={"appmetrica_device_id":{"Id":"798DFFE9-10A8-444E-B69B-0193E2B0EB2B"},"appmetrica_uuid":{"Id":"8b12d18676e548ee9e8f563fc292931b"},"appmetrica_device_id_hash":{"Id":"17421573730961678872"}}
Nov  7 12:01:10 Sandbox-03s-iPhone PixelArt(UnityFramework)[2821] <Notice>: AppMetricaController-> _DebugDeviceID: reason=
Nov  7 12:01:10 Sandbox-03s-iPhone PixelArt(AppMetricaLog)[2821] <Error>: -[AMADatabaseIntegrityProcessor checkIntegrityIssuesForDatabase:report:]:50 DB integrity check found issues: (
)
Nov  7 12:01:16 Sandbox-03s-iPhone PixelArt(AppMetricaLog)[2821] <Error>: -[AMADatabaseIntegrityProcessor checkIntegrityIssuesForDatabase:report:]:50 DB integrity check found issues: (
)
```
